tests/test_pynamodb.py

```diff
@@ -0,0 +1,78 @@
+# Base Tutorial: https://pynamodb.readthedocs.io/en/stable/tutorial.html
```

Comment:
would you create a test for DbUser similar to this?

Reply:
I'm planning to create tests for the permanent tables via the api endpoint test files and not exactly like this. The purpose this file served was getting familiar to the PynamoDB ORM, verifying that all CRUD ops work with pytest assertions, and existing as a reference for the syntax for future tests. Should I consider creating a file like this for new tables like DbUser, or should I continue with the plan I stated initially?

Reply:
yep makes sense. go for api tests